ui/src/fragments/Icons/Loading.svelte

```diff
@@ -1,22 +1,15 @@
-<!-- By Sam Herbert (@sherb), for everyone. More @ http://goo.gl/7AJzbL -->
 <svg
-  viewBox="0 0 38 38"
+  viewBox="0 0 24 24"
   xmlns="http://www.w3.org/2000/svg"
-  stroke="currentColor"
+  class="dark:stroke-white"
```

Comment:
please don't add colours direct to SVGs, just use `currentColor` as elsewhere and set the color in the parent container